src/components/NoteCard.css

```diff
@@ -41,3 +42,65 @@
   right: 4px;
   justify-content: end;
 }
+.overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 10;
+}
+.edit-note-container {
+  background: var(--white);
+  /* position: absolute; */
+  /* width: 50%; */
+  padding: 0.5rem 0.5rem 2rem;
+  /* top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%); */
+}
+.color-pallete {
+  background-color: var(--white);
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  gap: 4px;
+  width: 19%;
+  flex-wrap: wrap;
+  padding: 8px 12px;
+  box-shadow: 0 1px 2px 0 rgb(60 64 67 / 30%), 0 2px 6px 2px rgb(60 64 67 / 15%);
+}
+.color-item {
+  height: 30px;
+  width: 30px;
+  border: 1px solid var(--grey-2);
+}
+.bgffffff {
+  background-color: #ffffff;
+}
```

Comment:
colors you should use via variables everytime here

Reply:
I am not using these colors again that's why have not used variables here.

Reply:
Okay